### PR TITLE
Style updates

### PR DESCRIPTION
**Pull request by JakeM93**
Style updates based on feedback

### CODE REVIEW

**Comment by JakeM93** [comment]
gonne review each change in here first, this can always be changed, easier on the eyes

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -11,3 +11,9 @@
 // Your CSS partials
 @import "components/index";
 @import "pages/index";
+
+// global styling
+
+.body-dark-theme {
```
Comment (JakeM93) [inline app/assets/stylesheets/application.scss:17]: Feedback to make the background a little lighter for better contrast
I took inspiration from GitHub dark theme colours

File: app/assets/stylesheets/components/_button.scss [hunk 2]
```diff
@@ -16,3 +16,25 @@
   background: #1BCB7F;
   color: white;
 }
+.action-btn-large{
```
Comment (JakeM93) [inline app/assets/stylesheets/components/_button.scss:19]: Added some styling to the book, edit, view, delete buttons.
Have also added some color on the hover and active states
image
image

File: app/assets/stylesheets/components/_card.scss [hunk 3]
```diff
@@ -1,4 +1,6 @@
 .our-card {
-  color: #0b0c10;
-  height: 360px;
+  color: #ffffff;
```
Comment (JakeM93) [inline app/assets/stylesheets/components/_card.scss:2]: cars#index car colour so hat stands out and catches the eye
image

File: app/assets/stylesheets/components/_content.scss [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+@media only screen and (max-width: 1000px) {
+  .car-show-map {
+      display: none;
+  }
+}
+
+// used in my_cars and my_bookings
+.list-body-dark {
+  background-color: $dark-violet;
+}
```
Comment (JakeM93) [inline app/assets/stylesheets/components/_content.scss:23]: containers dark theme, to make contrast better with a darker background, based on feedback.
Took inspiration from this site - https://dribbble.com/shots/9836303-Text-field-UI-design-templates-S8-Figma-UI-kit

Also prettified the content boxes to have rounded boards and fix content centre.
image
image
image

File: app/assets/stylesheets/components/_navbar.scss [hunk 5]
```diff
@@ -2,6 +2,8 @@
   justify-content: space-between;
   font-size: 18px;
   padding: 18px;
+  width: 100%;
```
Comment (JakeM93) [inline app/assets/stylesheets/components/_navbar.scss:5]: Make the navbar full width of the page, and have a background colour (currently transparent)
Guess could add some overlay to make it slightly darker or lighter to see the seperation

File: app/views/cars/my_cars.html.erb [hunk 6]
```diff
@@ -23,10 +23,10 @@
...
           </div>
           <div class="col-12">
-            <div class="row justify-content-around">
-              <div class="list-button"><a href="/cars/<%=car.id%>"><i class="far fa-eye"></i></a></div>
-              <div class="list-button"><a href="/cars/<%=car.id%>/edit"><i class="far fa-edit"></i></a></div>
-              <div class="list-button"><a href="/cars/<%=car.id%>/destroy"><i class="far fa-trash-alt"></i></a></div>
+            <div class="row justify-content-around align-content-center">
+              <div class="list-button"><a href="/cars/<%=car.id%>"><i class="far fa-eye book-btn my-pages-action-btn"></i></a></div>
+              <div class="list-button"><a href="/cars/<%=car.id%>/edit"><i class="far fa-edit edit-btn my-pages-action-btn"></i></a></div>
+              <div class="list-button"><a href="/cars/<%=car.id%>/destroy"><i class="far fa-trash-alt delete-btn my-pages-action-btn"></i></a></div>
```
Comment (JakeM93) [inline app/views/cars/my_cars.html.erb:29]: Style updates

File: app/views/layouts/application.html.erb [hunk 7]
```diff
@@ -12,7 +12,7 @@
     <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload', defer: true %>
   </head>
 
-  <body>
+  <body class="body-dark-theme">
```
Comment (JakeM93) [inline app/views/layouts/application.html.erb:15]: assign dark theme class to body

File: db/seeds.rb [hunk 8]
```diff
@@ -29,7 +29,7 @@
       reg_number: Faker::Vehicle.license_plate,
       price: 500.00,
       user_id: test_account.id,
-      description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.",
+      description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry.",
```
Comment (JakeM93) [inline db/seeds.rb:32]: Reduced the description size

File: db/seeds.rb [hunk 9]
```diff
@@ -54,7 +54,7 @@
 ## Random Generated Data 
 
 fake_data_id = 0
-50.times do
+25.times do
```
Comment (JakeM93) [inline db/seeds.rb:57]: reduce the amount of fake data, improve Heroku deploy time

**Comment by JakeM93** [comment]
PR is lots of little changes mainly for the styling changes, that can also be changed, ust making a start

File: app/views/cars/index.html.erb [hunk 10]
```diff
@@ -4,20 +4,32 @@
...
             <h5 class="card-title"><%= "#{car.make} - #{car.model}" %></h5>
-            <p class="card-text"><%=car.description%></p>
+          </div>
+          <div class="card-body p-0">
+            <div class="row">
+              <div class="col">
+                <ul class="m-0" style="padding:0;text-align-left">
+                  <li class="card-text card-car-detail">£<%= car.price %> per day</li>
+                  <li class="card-text card-car-detail"><%= car.seats %> <i class="fas fa-chair"></i></li>
+                  <li class="card-text card-car-detail"><%= car.location%></li>
```
Comment (JakeM93) [inline app/views/cars/index.html.erb:22]: Removed description and show key information
Also added star review as a placeholder where we can put an average rating count?

image

File: app/views/cars/show.html.erb [hunk 11]
```diff
@@ -22,21 +22,22 @@
...
     </div>
-    <div class="col" style="min-width:300px;">
-      <div class="card-body">
+    <div class="col-xs-1 col-l-6 m-auto shadow" style="min-width:300px;">
+      <div class="card-body text-container-dark">
         <h5 class="card-title"><%= "#{@car.make} - #{@car.model}" %></h5>
         <p class="card-text"><%=@car.description%></p>
         <p class="card-text">£<%= @car.price %> per day</p>
-        <p class="card-text"><%= @car.seats %> seats</p>
+        <p class="card-text"><%= @car.seats %> <i class="fas fa-chair"></i></p>
```
Comment (JakeM93) [inline app/views/cars/show.html.erb:33]: Adding seats icon on the cars#show page

File: db/seeds.rb [hunk 12]
```diff
@@ -25,11 +27,11 @@
     puts "Creating cars"
     test_car = Car.create(model: Faker::Vehicle.model,
       make: Faker::Vehicle.manufacture,
-      location: Faker::Address.city,
+      location: Faker::Address.postcode,
```
Comment (JakeM93) [inline db/seeds.rb:30]: Changed faker car address to postcode to be used for getting more accurate location for maps

**Review by directtoanne — APPROVED** [review]
good

**Pull request merged**